circuits/voteverifier/artifacts.go

```diff
@@ -0,0 +1,17 @@
+package voteverifier
+
+import (
+	"github.com/vocdoni/vocdoni-z-sandbox/circuits"
+	"github.com/vocdoni/vocdoni-z-sandbox/types"
+)
+
+var Artifacts = circuits.NewCircuitArtifacts(
+	&circuits.Artifact{
+		RemoteURL: "https://media.githubusercontent.com/media/vocdoni/vocdoni-circuits-artifacts/main/voteverifier/voteverifier.pk",
```

Comment:
I think all these constants (which are very relevant) must be defined together in a specific package. Maybe /config so we have everything centralized there.